core/frontend-devtools/src/tools/RealityModelTools.ts

```diff
@@ -81,8 +81,12 @@ export class SaveRealityModelTool extends Tool {
 }
 
 function changeRealityModelAppearanceOverrides(vp: Viewport, overrides: FeatureAppearanceProps, index: number): boolean {
-  const existingOverrides = vp.getRealityModelAppearanceOverride(index);
-  return vp.overrideRealityModelAppearance(index, existingOverrides ? existingOverrides.clone(overrides) : FeatureAppearance.fromJSON(overrides));
+  const model = vp.displayStyle.settings.contextRealityModels.models[index];
+  if (!model)

```

Comment:
The viewport listener now handles the invalidating render plan etc. so the viewport methods are superfluous, right?

Reply:
Correct. Viewport had ones that called DisplayStyleState ones that called DisplayStyleSettings ones...I removed the redundant non-`public` ones.